package.json

```diff
@@ -5,7 +5,7 @@
     "backbone": "~1.3.2",
     "coffee-script": "1.6.1",
     "edx-pattern-library": "~0.12.4",
-    "edx-ui-toolkit": "~0.9.1",
+    "edx-ui-toolkit": "git+https://github.com/edx/edx-ui-toolkit.git#16626810f74294125b5dc0fbb0b19f713b40afdc",
```

Comment:
Don't forget that you can't merge this change, but should instead do a UI Toolkit release and then reference the new version number.